tests/test_core.py

```diff
@@ -35,6 +36,11 @@ def no_wasm(f):
 def no_wasm_backend(f):
   return skip_if(f, 'is_wasm_backend')
 
+def no_wasm_backend_note(note=''):
```

Comment:
Yes but the implementation is more complex. We have to take either-a-note-or-a-function, and then switch based on what the argument is to either indirecting one or two more times.
I prefer form in the rest of the code however so it's probably worth keeping the complexity in the decorator itself. I'll make the change.